fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -192,6 +192,11 @@ public static ConstantOperator date(ConstantOperator arg) {
         return ConstantOperator.createDate(datetime);
     }
 
+    @FEFunction(name = "unix_timestamp", argTypes = {}, returnType = "INT")
+    public static ConstantOperator unixTimestampNow() {
```

Comment:
I have a question about this.
When there are many now() function call in one SQL, these now() will generate different values if we treat it like this.
And now we create a `now()` in global values, I think we should keep consistent with it.

Reply:
I also have this question. Could that `unixtimestamp()` be rewritten to a constant value when delivering the plan to BE?

Reply:
> I have a question about this. When there are many now() function call in one SQL, these now() will generate different values if we treat it like this. And now we create a `now()` in global values, I think we should keep consistent with it.

Now same expression will analyze to one instance in project node, can be considered the result of `now()` in an SQL is all same. Of course, some special cases may not be handled

Reply:
So, I think we should treat all `now()` in the same format.
If `now()` is in some expression which is not const, it will be computed in backend.
We should handle such case.

Reply:
`now()` in an SQL must always computed in frontend now, my means is the `now()` will compute twice maybe in some special clause or stynax, it's can be improve later.

Reply:
If this PR was merged, would some special case generate wrong result?

Reply:
I think will, this PR just will make `unix_timestamp()` always computed in frontend to optimize query preformance,  it's can't resolve multi-compute function(`now()/unix_timestamp()`) in an SQL. Your question need add global values in optimizer to resolve.

Reply:
@Seaven 
Thanks.
I see, could you create an issue to record this problem and fix it later?

Reply:
Issue #1566